lingua_franca/lang/parse_en.py

```diff
@@ -400,7 +409,7 @@ def _extract_whole_number_with_text_en(tokens, short_scale, ordinals):
                 current_val = val
 
         else:
-            if all([
+            if current_val and all([
```

Comment:
this is an unrelated minor bug fix, during testing i ran into a case where current_val was None and int comparison inside all() would throw an exception